aws-cdn-templates/one-cdn.yaml

```diff
@@ -63,16 +63,10 @@ Parameters:
 
   S3LogsBucket:
     Type: String
-    Default: '-'
+    Default: "-"
     Description: Access Log bucket name
 
 Conditions:
-  # Is landing site if name startsWith 'web-landing-cdn-'
-  IsLanding:
```

Comment:
IsLanding is used around the file. Adjust where it is used

Reply:
done